pkg/kube/kube.go

```diff
@@ -26,12 +26,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License
 
-package helm
+package kube
```

Comment:
Ouch, I missed that one !

Reply:
Hello @aslafy-z

Thank you for your feedbacks but I think not understand the problem, sorry.

The go files in the `api` subdirectory have as the package name `api` : https://github.com/FairwindsOps/pluto/blob/master/pkg/api/columns.go#L15

The go files in the `discovery-api` subdirectory have as the package name `discoveryapi` : https://github.com/FairwindsOps/pluto/blob/master/pkg/discovery-api/discovery_api.go#L29

The go files in the `finder` subdirectory have as the package name `finder` : https://github.com/FairwindsOps/pluto/blob/master/pkg/finder/finder.go#L29

The go files in the `helm` subdirectory have as the package name `helm` : https://github.com/FairwindsOps/pluto/blob/master/pkg/helm/helm.go#L29

Regarding for the import of a package in a module, the subdirectory is used.
Here is the golang convention : https://go.dev/ref/mod#glos-package-path

Reply:
In my last pr, I moved this file to the `kube` directory but forgot to update the package name at the top of the file, you just fixed my mistake in this pr. Thank you for fixing my mess here! I was wondering how the import could work without this value matching the directory name

Reply:
I just got it :-)
Ultimately it's not a problem.
Thank you for the feature of being able to specify a namespace with the discovery api subcommand, it saves our lives :-)